.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "semi": true,
+  "trailingComma": "all",
+  "singleQuote": true,
+  "printWidth": 100,
+  "tabWidth": 2
+}
```

Comment:
while we're at it, can we add an end of line rule like this:
```
"lf": auto
```
https://prettier.io/docs/en/options.html#end-of-line

Reply:
Okay will do that now